identity-apps-core/apps/recovery-portal/src/main/webapp/password-reset.jsp

```diff
@@ -49,7 +49,7 @@
     String callback = (String) request.getAttribute(IdentityManagementEndpointConstants.CALLBACK);
     String username = request.getParameter("username");
     String userStoreDomain = request.getParameter("userstoredomain");
-    String type = request.getParameter("type");
+    String type = Encode.forJava(request.getParameter("type"));
     String orgId = request.getParameter("orgid");
     String spId = request.getParameter("spId");
```

Comment:
Don't we need to encode these params as well?

Reply:
Encoding only the params and places reported